drivers/spi/spi_ll_stm32.h

```diff
@@ -183,6 +186,17 @@ static inline void ll_func_disable_int_errors(SPI_TypeDef *spi)
 #endif /* st_stm32h7_spi */
 }
 
+static inline bool ll_func_are_int_disabled(SPI_TypeDef *spi)
+{
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi)
+	return (spi->IER == 0U);
+#else
+	return ((LL_SPI_IsEnabledIT_ERR(spi) ||
+		 LL_SPI_IsEnabledIT_RXNE(spi) ||
+		 LL_SPI_IsEnabledIT_TXE(spi)) == 0);
```

Comment:
I would rather suggest a form easier to read:
```suggestion
	return !LL_SPI_IsEnabledIT_ERR(spi) &&
	       !LL_SPI_IsEnabledIT_RXNE(spi) &&
	       !LL_SPI_IsEnabledIT_TXE(spi);